### ISSUE TITLE
Alias fails for domains with `_` in name

### ISSUE THREAD

**Issue by domderen** [zeit/now-issues#4]
Hey,

I tried to alias my `now` deployment to my custom domain with the following command:

```bash
now alias https://app-name-fbontxfmdb.now.sh deploy_per_branch.app.my-domain.com
```
and it failed with the following error:

```bash
deploy_per_branch.app.my-domain.com is a custom domain.
Verifying the DNS settings for deploy_per_branch.app.my-domain.com (see https://zeit.world for help)
Verification OK!
Provisioning certificate for deploy_per_branch.app.my-domain.com
Error! Unknown error: Error
    at r.<anonymous> (/Users/dominik/.nvm/versions/node/v6.3.1/lib/node_modules/now/build/lib/alias.js:1:13699)
    at tryCatch (/Users/dominik/.nvm/versions/node/v6.3.1/lib/node_modules/now/node_modules/babel-runtime/regenerator/runtime.js:88:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (/Users/dominik/.nvm/versions/node/v6.3.1/lib/node_modules/now/node_modules/babel-runtime/regenerator/runtime.js:341:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (/Users/dominik/.nvm/versions/node/v6.3.1/lib/node_modules/now/node_modules/babel-runtime/regenerator/runtime.js:121:21)
    at step (/Users/dominik/.nvm/versions/node/v6.3.1/lib/node_modules/now/node_modules/babel-runtime/helpers/asyncToGenerator.js:17:30)
    at /Users/dominik/.nvm/versions/node/v6.3.1/lib/node_modules/now/node_modules/babel-runtime/helpers/asyncToGenerator.js:28:13
    at process._tickCallback (internal/process/next_tick.js:103:7)
```

When I changed my custom domain and substituted the `_` character with `-`, it worked just fine. I assume there is a problem with provisioning a certificate for domain names that contain `_` in them.

**Comment by nkzawa** [comment]
`_` is not valid for hostname, so it has to be `-` ? @rauchg @rase- 
I think the error message can be improved at least.

**Comment by onbjerg** [comment]
@nkzawa [RFC 2181](http://www.ietf.org/rfc/rfc2181.txt), section 11, "Name syntax":
> The DNS itself places only one restriction on the particular labels that can be used to identify resource records. That one restriction relates to the length of the label and the full name. [...] Implementations of the DNS protocols must not place any restrictions on the labels that can be used. In particular, DNS servers must not refuse to serve a zone because it contains labels that might not be acceptable to some DNS client programs.

I think you mean _host names_ in which only letters, digits and hyphens are allowed. ([RFC 1123](http://www.ietf.org/rfc/rfc1123.txt), section 2.1 "Host Names and Numbers")

**ELI5**: Domain names can have underscores and all that, but host names can not. @domderen's alias is valid.

**Comment by leo** [comment]
https://github.com/zeit/now/issues/17
